src/Util/Network.php

```diff
@@ -570,10 +569,8 @@ public static function isUrlBlocked($url)
 			return false;
 		}
 
-		$host = strtolower($h['host']);
```

Comment:
And this extra can then be avoided (which you did here) only because of using 2nd parameter with `PHP_URL_HOST`). Still I would love to replace all these `parse_url()` invocations to a class.